.bluemix/pipeline.yml

```diff
@@ -36,24 +36,24 @@ stages:
     script: |
       #!/bin/bash
       # Push app
-      if ! cf app $CF_APP; then  
-        cf push $CF_APP
+      if ! cf app "$CF_APP"; then  
+        cf push "$CF_APP"
       else
-        OLD_CF_APP=${CF_APP}-OLD-$(date +"%s")
+        OLD_CF_APP="${CF_APP}-OLD-$(date +"%s")"
         rollback() {
           set +e  
-          if cf app $OLD_CF_APP; then
-            cf logs $CF_APP --recent
-            cf delete $CF_APP -f
-            cf rename $OLD_CF_APP $CF_APP
+          if cf app "$OLD_CF_APP"; then
+            cf logs "$CF_AP"P --recent
```

Comment:
small typo - 2nd P should be inside the quotes

Reply:
good catch.  I did run the script and it worked for a new app.  i didn't test the existing app.  let me fix/test

Reply:
@jerh616.  fixed, confirmed its working